projects/ui/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "ui",
   "version": "2.1.0",
+  "description": "x",
```

Comment:
x?

Reply:
Updated in my commit with metadata above

Reply:
Actually, @silochad already updated.